geo/src/algorithm/coordinate_position.rs

```diff
@@ -371,21 +366,30 @@ where
     // See: https://en.wikipedia.org/wiki/Point_in_polygon#Winding_number_algorithm
     let mut winding_number = 0;
     for line in linestring.lines() {
+        // Edge Crossing Rules:
+        //   1. an upward edge includes its starting endpoint, and excludes its final endpoint;
+        //   2. a downward edge excludes its starting endpoint, and includes its final endpoint;
+        //   3. horizontal edges are excluded
+        //   4. the edge-ray intersection point must be strictly right of the point P.
```

Comment:
What is "the point P" referring to in these notes?

Reply:
The point to test for intersection/location/inclusion.

Reply:
So can we call it "`coord`" to match with the code it's documenting - we don't use `P` anywhere else in the docs or code.

Reply:
yes, done!